packages/strapi-plugin-content-manager/services/ContentManager.js

```diff
@@ -73,12 +73,21 @@ module.exports = {
   add: async (params, values, source) => {
     // Multipart/form-data.
     if (values.hasOwnProperty('fields') && values.hasOwnProperty('files')) {
+      // get model attributes
+      const { attributes: modelAttibutes } = strapi.models[params.model];
```

Comment:
This code won't work for moels that aren't in the strapi.models global (e.g plugin models)

Reply:
Indeed!
Can the content-manager plugin be used in other plugins?
Can you provide a specific use case and/or example?

I'm just trying to understand the issue as it seems this plugin is only used in the context of user-created API models, so available in `strapi.models`.
Maybe I'm missing something. :)

Reply:
essentially (the way I understand it) the AdminUI sends everything as a string to the backend. Though I could be wrong

Reply:
for example the administrators are editable through the content-manager and this model isn't ins tarpi.models. the same goes for users even though it goes through the routing policy and get forward to the user controller directly your parsing needs to also be added to the routing policy in the content-manager plugin. 

The form data does send all the data as strings because it handles images. 
Be awar that this code will change in an upcoming release.